src/ER_V/acl/acl.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+#include <stddef.h>
+
+#include "acl/acl.h"
+#include "acl/acl_private.h"
+#include "log/log.h"
+
+#define LOG_CONSOLE_THRESHOLD_THIS  LOG_THRESHOLD_MAX
+#define LOG_FILE_THRESHOLD_THIS     LOG_THRESHOLD_MAX
+
+
+ACL_Resources resources;
+
+int ACL_Command_init(ACL_Command *cmd)
+{
+    if (!cmd) STD_FAIL;
+
+    memset(cmd, 0, sizeof(ACL_Command));
+    
+    DATA_S_List_Node_init(&cmd->node);
+    DATA_S_List_append(&resources.queue, &cmd->node);
+
+    return 1;
+}
+
+int ACL_init()
+{
+    for(uint16_t iter = 0; iter < ACL_CMD_COUNT; iter++)
+    {
+        ACL_Command_init(&resources.cmd_pool[iter]);
+    }
+
+    return 1;
+}
+
+int ACL_calc_enqueue_shift_command(ACL_Command *cmd, ACL_Axis axis, float degree_count)
+{
+    if (!cmd) STD_FAIL;
+
+    int encoder_count;
+
+    if (axis == BASE_AXIS)
```

Comment:
This could probably be better implemented as a switch statement, with default calling STD_FAIL

Reply:
maybe having a local variable called "conversion", with the switch statement putting the appropriate value in. I also can show you a kinda hacky way to do it without any conditions, but it veers from "good practice".